sites/public/pages/index.tsx

```diff
@@ -80,11 +81,18 @@ export default function Home(props: IndexProps) {
 export async function getStaticProps() {
   let listings = []
   try {
-    // const response = await axios.get(process.env.listingServiceUrl)
-    const response = await axios.get(
-      process.env.listingServiceUrl +
-        "?view=base&limit=all&filter[$comparison]=<>&filter[status]=pending"
-    )
+    const response = await axios.get(process.env.listingServiceUrl, {
+      params: {
+        view: "base",
+        limit: "all",
+        $comparison: "<>",
+        status: "pending",
+      },
+      paramsSerializer: (params) => {
+        return qs.stringify(params)
+      },
+    })
+
```

Comment:
And again here, shouldn't there be a filter key that's an array of object? I tested with setting filter: [with different objects here] and it worked.